src/NATS.Client.Core/NatsConnection.RequestReply.cs

```diff
@@ -61,6 +64,42 @@ public partial class NatsConnection
         }
     }
 
+    [SkipLocalsInit]
+    private static string NewInbox(ReadOnlySpan<char> prefix)
+    {
+        Span<char> buffer = stackalloc char[64];
+        var separatorLength = prefix.Length > 0 ? 1u : 0u;
```

Comment:
I'm not sure if support for empty prefixes is really needed, but right now nothing is preventing users from doing it.

Reply:
AFAIK, practice is to have a prefix so the accounts can be permissined accordingly.

Reply:
fwiw, Actually just realized an empty prefix might create interesting behaviours in subscription manager i.e. `SubscriptionManager.IsInboxSubject()` would always return `true` causing all subscriptions to go through an internal muxed 'inbox' subscription.